src/services/location.js

```diff
@@ -221,37 +305,37 @@ ngeo.Location.prototype.getParamKeysWithPrefix = function(prefix) {
  * @export
  */
 ngeo.Location.prototype.getFragmentParamKeysWithPrefix = function(prefix) {
-  return this.getFragmentParamKeys().filter(key => key.indexOf(prefix) == 0);
+  const keys = [];
```

Comment:
Why not but it's not easy, we should be able to do generic typing to avoid casting.
We filter the key than we should do tow for ...